coremltools/converters/mil/mil/ops/defs/reduction.py

```diff
@@ -117,6 +117,36 @@ def type_inference(self):
 
 @register_op(doc_str="TODO")
 class reduce_argmax(reduce_arg):
+    """
+    Computes the indices of the maximum value across dimensions of a tensor.
+    In case of ties, the identity of the return value is not guaranteed.
+    
+    Parameters
+    ----------
+    x: <*,T> (Required)
+        * Must be 1-dimensional or higher.
+    
+    axis: const<i32> (Optional)
+        * The dimension to reduce. Default is ``-1``.
+    
+    keep_dims: const<bool> (Optional, default=False)
+        * If ``False``, the rank is reduced for each entry in ``axis``.
```

Comment:
---> If ``False``, the rank is reduced by 1, by removing the dimensions specified via ``axis``.